learn/learn_runner.go

```diff
@@ -120,8 +107,10 @@ func createHARFile(in <-chan akinet.ParsedNetworkTraffic, interfaceName string,
 
 	l := har.NewLogger()
 	for t := range in {
-		if containsCLITraffic(t.Content) {
-			continue
+		if util.ContainsCLITraffic(t) {
+			if !viper.GetBool("dogfood") {
```

Comment:
Flip these conditions? Checking the boolean seems cheaper than checking `ContainsCLITraffic`.

Reply:
Probably, but for non-Akita users the situation is reversed: the GetBool is strictly overhead, so it should be executed as few times as possible.

Reply:
Wouldn't branch prediction take care of that overhead? But, yes, we shouldn't be optimizing for dogfooding.